src/t8_schemes/t8_default/t8_default_vertex/t8_default_vertex.cxx

```diff
@@ -22,11 +22,80 @@
 
 #include <t8_schemes/t8_default/t8_default_common/t8_default_common.hxx>
 #include <t8_schemes/t8_default/t8_default_vertex/t8_default_vertex.hxx>
-#include <t8_schemes/t8_default/t8_default_vertex/t8_dvertex_bits.h>
+
+void
+t8_dvertex_init_linear_id (t8_dvertex_t *v, int level, t8_linearidx_t id)
+{
+  T8_ASSERT (0 <= level && level <= T8_DVERTEX_MAXLEVEL);
+  T8_ASSERT (0 == id);
+
+  /* Set the level */
+  v->level = level;
+}
 
 /* We want to export the whole implementation to be callable from "C" */
 T8_EXTERN_C_BEGIN ();
```

Comment:
```suggestion
```
I don't think that this is necessary anymore, since we switched to the CRTP implementation

Reply:
Please also remove the `T8_EXTERN_C_END ();` at the end of the file.